apps/client/package.json

```
@@ -37,6 +37,7 @@
     "@remix-run/serve": "2.4.1",
     "@tailwindcss/forms": "0.5.7",
     "@tailwindcss/typography": "0.5.10",
+    "framer-motion": "^10.15.2",
```

```suggestion
    "framer-motion": "10.15.2",
```

Nit:

We'd wanna lock deps so that these packages don't update themselves 👍🏾

That said, we only update our packages manually.